spec/rails_helper.rb

```diff
@@ -5,6 +5,8 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'coveralls'
```

Comment:
these two lines need to be on top of the file

Reply:
done!